.gitignore

```
@@ -6,3 +6,6 @@ __pycache__
 
 # Editor
 .vscode
+
+# Tests
+*.json
```

Just for my mental health lol
```suggestion
*.json

```